src/osd/osd_types.cc

```diff
@@ -5085,7 +5085,7 @@ void ScrubMap::encode(bufferlist& bl) const
 
 void ScrubMap::decode(bufferlist::iterator& bl, int64_t pool)
 {
-  DECODE_START_LEGACY_COMPAT_LEN(3, 2, 2, bl);
+  DECODE_START_LEGACY_COMPAT_LEN(3, 3, 3, bl);
```

Comment:
@athanatos Version 2 is so old, it won't be an issue anymore, so I removed code for version 2 compatibility that would need to be updated with your change.